activiti-spring-boot-starter/src/test/resources/processes/JobEventsTest.testJobCanceledEventOnBoundaryEvent.bpmn20.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you mind to rename this file to something like `ProcessWithBoundaryTimer`?

Reply:
Renamed: https://github.com/Activiti/Activiti/pull/2814/commits/ee5aa99d2e9c2e99b10557e2e4ce7b38a9de2e19